build fails when linking libocca.dylib with Metal

When building with `./configure-cmake.sh` on macOS 13.3:
```
[ 64%] Linking CXX shared library lib/libocca.dylib
ld: warning: directory not found for option '-F/Library/Developer/CommandLineTools/SDKs/MacOSX13.1.sdk/System/Library/Frameworks/Metal.framework /Library/Developer/CommandLineTools/SDKs/MacOSX13.1.sdk/System/Library/Frameworks/CoreServices.framework /Library/Developer/CommandLineTools/SDKs/MacOSX13.1.sdk/System/Library/Frameworks'
Undefined symbols for architecture arm64:
  "_MTLCopyAllDevices", referenced from:
      occa::api::metal::getDeviceCount() in metal.mm.o
      occa::api::metal::getDevice(int) in metal.mm.o
ld: symbol(s) not found for architecture arm64
clang: error: linker command failed with exit code 1 (use -v to see invocation)
make[2]: *** [lib/libocca.dylib] Error 1
make[1]: *** [CMakeFiles/libocca.dir/all] Error 2
make: *** [all] Error 2
```

--- a/cmake/FindMETAL.cmake
+++ b/cmake/FindMETAL.cmake
@@ -20,7 +20,5 @@ if(METAL_FOUND AND NOT TARGET OCCA::depends::METAL)
   # Create our wrapper imported target
   # Put it in the OCCA namespace to make it clear that we created it.
   add_library(OCCA::depends::METAL INTERFACE IMPORTED)
-  set_target_properties(OCCA::depends::METAL PROPERTIES
-    INTERFACE_LINK_LIBRARIES "${METAL_LIBRARY} ${CORE_SERVICES} ${APP_KIT}"
-  )
+  target_link_libraries(OCCA::depends::METAL INTERFACE ${METAL_LIBRARY} ${CORE_SERVICES} ${APP_KIT})
 endif()

